frontend/amundsen_application/static/js/pages/TableDetailPage/index.tsx

```diff
@@ -401,12 +410,12 @@ export class TableDetail extends React.Component<
           </header>
           <div className="column-layout-1">
             <aside className="left-panel">
-              {!!tableNotice && (
+              {notices.map((notice) => (
```

Comment:
What if notices is empty?

At this point, I would extract a component here, something like <NoticeList /> and move all the logic inside.